### DISCUSSION TITLE
Meeting 2025-01-07

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#810] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2025-01-07)

**Comment by adamgreig** [comment]
* https://github.com/rust-embedded/cortex-m/issues/572 HardFault linking too far for `b` on armv6
* https://github.com/rust-embedded/cortex-m/issues/571 semihosting sending 4 bytes per char?
* https://github.com/rust-embedded/cortex-m/issues/570 issues with cortex-m LR register access

**Comment by jannic** [comment]
Invitation list for embedded wg room at RustWeek unconf / all hands meeting.

**Reply by BartMassey** [reply]
I would like to be included.
